Server: Error during "npm start" on Ubuntu 18.04.5 LTS

Following error occurs, when trying to start the server on Ubuntu 18.04.5 LTS:
```
> server@0.1.0 start /home/lennart/Documents/projects/neomdb/server
> ts-node-dev --respawn --transpileOnly ./src/server.ts

ts-node-dev: no script to run provided
Usage: ts-node-dev [options] script [arguments]

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! server@0.1.0 start: `ts-node-dev --respawn --transpileOnly ./src/server.ts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the server@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/server/package.json
+++ b/server/package.json
@@ -22,7 +22,7 @@
   },
   "scripts": {
     "build": "tsc",
-    "start": "ts-node-dev --respawn --transpileOnly ./src/server.ts",
+    "start": "ts-node-dev --respawn --transpile-only ./src/server.ts",
     "prod": "npm run build && npm run start",
     "lint": "tslint --project ./"
   }

